File: Beginner/Exer. 1003/main.java
import java.util.Scanner;

class Main {
    public static int sum(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] num = new int[2];
        num[0] = scanner.nextInt();
        num[1] = scanner.nextInt();
        scanner.close();

        System.out.println("SOMA = " + sum(num[0], num[1]));
    }
}

File: Beginner/Exer. 1003/main.c
#include <stdio.h>

int sum(int A, int B) {
    return A + B;
}

int main(int argc, char *argv[]) {
    int num[2];
    scanf("%d", &num[0]);
    scanf("%d", &num[1]);
    
    printf("SOMA = %d\n", sum(num[0], num[1]));
    return 0;
}